moveit_plugins/moveit_ros_control_interface/README.md

```diff
@@ -22,12 +22,12 @@ In your MoveIt! launch file (e.g. `ROBOT_moveit_config/launch/ROBOT_moveit_contr
 <arg name="moveit_controller_manager" default="moveit_ros_control_interface::MoveItControllerManager" />
 ```
 
-And make sure so set the `ros_control_namespace` parameter to the namespace (without the /contoller_manager/ part) of the ros_control-based node you like to interface.
-If you are using the `moveit_setup_assistent` you can add it to `ROBOT_moveit_config/config/ROBOT_controllers.yaml`, e.g.:
+And make sure to set the `ros_control_namespace` parameter to the namespace (without the /controller_manager/ part) of the ros_control-based node you like to interface.
+If you are using the `moveit_setup_assistent` you can add it to `ROBOT_moveit_config/config/controllers.yaml`, e.g.:
 ```
-ros_control_namespace: /ROBOT
+ros_control_namespace: /ROS_CONTROL_NODE
 controller_list:
-  - name: /ROBOT/position_trajectory_controller
+  - name: /ROS_CONTROL_NODE/position_trajectory_controller
```

Comment:
For baxter I used ``/robot/right_position_trajectory_controller`` because that is how Rethink names it, and for an IIWA project I used `` /iiwa_7_r800/reflexxes_trajectory_controller``. I'm pretty sure ROBOT is good here, though I also think that is typically the same name as ROS_CONTROL_NODE. I vote to revert the name